src/build.d

```diff
@@ -1444,6 +1449,30 @@ class Dependency
             }
         }
     }
+
+    /// Writes relevant informations about this dependency to stdout
+    private void dump()
+    {
+        scope writer = stdout.lockingTextWriter;
+        void write(T)(string fmt, T what)
+        {
+            static if (is(T : bool))
+                bool print = what;
+            else
+                bool print = what.length != 0;
+
+            if (print)
+                writer.formattedWrite(fmt, what);
+        }
+
```

Comment:
Maybe print a delimiter to distinguish between operations
```D
write("--------------------------------------------------------------------------------\n");
write("This Operation Failed:\n");
```

Reply:
Added